software/qst.yml

```diff
@@ -0,0 +1,13 @@
+name: "QST"
+website_url: "https://qstonline.org"
+source_code_url: "https://sourceforge.net/projects/qstonline/"
+description: "QST, the worlds unparalleled open source online assessment software. From a quick quiz on your phone to large scale, high stakes, proctored desktop testing, we make it easy, secure and economical."
+licenses:
+  - GPL-2.0
+platforms:
+  - Perl
+tags:
+  - Learning and Courses
+  - Knowledge Management Tools
+depends_3rdparty: false
```

Comment:
```suggestion
```
As it's false by default, feel free to omit this line. Other then that, looks all good to me!